dbus-server/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM debian:buster
+
+RUN apt-get update && apt-get install -y dbus && rm -rf /var/lib/apt/lists/*
+
+CMD ["/usr/bin/dbus-daemon", "--system", "--address=tcp:host=0.0.0.0,port=12434", "--nofork", "--nopidfile", "--systemd-activation", "--syslog-only"]
```

Comment:
I'm not against doing it this way, but I would investigate the possibility of calling this directly inside `.gitlab-ci.yml` instead, so that we can avoid an extra Docker instance in local unit tests, and let them use the already available DBus server instead.

Reply:
Ack, in gitlab we'll use a local dbus daemon.